src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs

```diff
@@ -67,72 +65,12 @@ public static Runspace CreateRunspace(PSHost host)
                 throw PSTraceSource.NewArgumentNullException("host");
             }
 
-            return CreateRunspace(host, RunspaceConfiguration.Create());
+            return new LocalRunspace(host);
         }
```

Comment:
We should construct the InitialSessionState here and call the appropriate constructor.

As for using CreateDefault vs. CreateDefault2, the argument that CreateDefault2 is faster is apparently very wrong, I measure it as 8X slower:

```
#88 PS> 'CreateDefault','CreateDefault2' | %{  measure-command {
>> $iss = [InitialSessionState]::$_();
>> 1..100 | % { $ps = [powershell]::Create($iss).AddCommand('echo').AddArgument(1); $ps.Invoke(); $ps.Dispose() } } }

1.003s
8.247s
```

If I change the example to not load any commands, then they are more similar:

#89 PS> 'CreateDefault','CreateDefault2' | %{  measure-command {
>> $iss = [InitialSessionState]::$_();
>> 1..100 | % { $ps = [powershell]::Create($iss).AddScript('1'); $ps.Invoke(); $ps.Dispose() } } }

1.209s
1.037s

Reply:
will update

Reply:
On my MacBookPro, CreateDefault2() is 3x slower and on Windows it's 2.5x slower, but consistently slower to use CreateDefault2()